src/Legend.tsx

```diff
@@ -158,7 +158,7 @@ export function Legend({
         </label>
         <button
           disabled={!config?.monitorUISettings?.connected}
-          className="pause-button"
+          className={`pause-button ${pause ? "paused" : ""}`}
```

Comment:
Please use `classnames` like this:
```
import classNames from "classnames";
```

```suggestion
          className={classNames("pause-button", { paused: pause })}
```